docs/renewables/wind/rap.rst

```diff
@@ -0,0 +1,13 @@
+Rapid Refresh
+#############
+`RAP`_ (Rapid Refresh) is the continental-scale NOAA hourly-updated
+assimilation/modeling system operational at the National Centers for Environmental
+Prediction (NCEP). RAP covers North America and is comprised primarily of a numerical
+weather model and an analysis system to initialize that model. RAP provides, every hour
```

Comment:
RAP provides, every hour -> RAP provides, for every hour

Reply:
Done